prow/build-tests.sh

```diff
@@ -42,6 +42,7 @@ function create_gcb_env() {
   touch /workspace/manifest.txt
 
 cat << EOF > "/workspace/gcb_env.sh"
```

Comment:
This file better be renamed to release_test (along with the test job, which probably should be moved to post-submit instead of presubmit unless you plan to block PRs)